libs/model/src/models/collaboration.ts

```diff
@@ -55,12 +53,17 @@ export default (
       createdAt: 'created_at',
       updatedAt: 'updated_at',
       underscored: true,
-    },
+      noPrimaryKey: true,
```

Comment:
@timolegros this is the `noPrimaryKey` option I was mentioning, also used in memberships below.

Reply:
@Rotorsoft does this do anything?? It's not in the types and I can't find anything in the docs about this option.

Reply:
This is v7 stuff, removing.